web/core/local-db/worker/db.ts

```
@@ -58,7 +66,7 @@ export class DBClass {
   }
 
   runQuery(sql: string) {
-    return this.instance.exec(sql);
+    return this.instance?.exec && this.instance.exec(sql);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)